test/commands/view/heimdall-view-cli.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Test heimdall SAF CLI Command', () => {
 describe('Test Heimdall Embedded', () => {
   let server: Server
 
-  beforeEach(done => {
+  beforeEach((done) => {
```

Comment:
to make this PR easier to process, can we move all the linting / style changes to a different PR that can be merged more easily?  i feel like the actual important stuff in this PR is getting buried

Reply:
I understand your concern, I should have separated the linting move from @stylistic/eslint-plugi v3 to v4 on a separate PR and leave the updates for delta here. I really am running short on time as I go on PTOBish on Saturday. 

The issue was that when I pulled this PR dependabot had update the @stylistic/eslint-plugi  and I couldn't lint until I fixed the linting stuff.

The files changed related to the Delta updates are:

- delta.ts
- update_controls4delta.ts
- globals.ts
- artifacts.ts
- README
- package.json (added adm-zip, removed zip-lib)